test/tapeimage.cpp

```diff
@@ -454,13 +454,6 @@ TEST_CASE(
 
         err = lfp_seek(tif, 8);
         CHECK(err == LFP_OK);
-
```

Comment:
About "Relax tapeimage/underlying tell relationship"

I wonder if we shouldn't then remove all the bits and pieces in tests that compare with underlying tell.
We have these checks as parts of some tests ("Peek into the underlying protocol") and also as the tests themselves ("Tape image: multiple seeks and tells match underlying handle").

In practice tests still check many valid things as in majority of cases underlying tell is very much predictable.

But if we conceptually relax/remove the connection, then we either need strict rules for when underlying tell still should match (thus for example defined by tests), or do not test this at all.

Reply:
You raise a good point, and I don't have any good answer but "maybe".

As you correctly point out, they do test some things that _are_ indeed predictable, at least in the current implementation.

This is what I would propse: We keep them around (since they haven't broken yet), but are prepared to adjust and/or kill them the second they _do_ break, if anything to use them to flag whenever they are affected.